src/mtype.h

```diff
@@ -157,6 +157,8 @@ enum m_flag : int {
     MF_AVOID_DANGER_2,      // This monster will path around most dangers instead of through them.
     MF_PRIORITIZE_TARGETS,  // This monster will prioritize targets depending on their danger levels
     MF_NOT_HALLU,           // Monsters that will NOT appear when player's producing hallucinations
+	MF_MILKABLE,            // This monster is milkable.
+	MF_SIMPLEMINDED,		// This monster is simpleminded.
```

Comment:
The name is rather nondescriptive. How about `MF_PET_WONT_FOLLOW`?
Also, tabs - here and in other files.